index.bs

```diff
@@ -220,14 +220,21 @@ An [=/XR device=] has a <dfn>list of supported modes</dfn> (a [=/list=] of [=/st
 
 Each [=/XR device=] has a <dfn for="XR device">list of enabled features</dfn> for each {{XRSessionMode}} in its [=list of supported modes=], which is a [=/list=] of [=feature descriptors=] which MUST be initially an empty [=/list=].
 
+The user-agent has a <dfn>list of immersive XR devices</dfn> (a [=/list=] of [=/XR device=]), which MUST be initially an empty [=/list=]. 
```

Comment:
Do we want this and the `immersive XR device` to be marked as `for=""` like the other `dfn`'s in this section? Would require every reference to be changed to `[=/immersive XR device=]`, etc. but you're already stripping off the `XRSystem` anyway, so that shouldn't be a big deal.